valhalla/midgard/openlr.h

```diff
@@ -129,9 +129,10 @@ struct LocationReferencePoint {
                          unsigned char lfrcnp = 0)
       : longitude(!prev ? integer2decimal(decimal2integer(longitude))
                         : prev->longitude +
-                              (std::round((longitude - prev->longitude) * 100000) / 10000)),
```

Comment:
its hard to see, which is why i made the mistake but the last number there is missing a 0

Reply:
Perhaps make a const variable of it to remove magic constanst?